core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2312,10 +2313,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_TIERED_STORE_LEVEL0_DIRS_MEDIATYPE =
+  public static final PropertyKey WORKER_TIERED_STORE_LEVEL0_DIRS_MEDIUMTYPE =
       new Builder(Template.WORKER_TIERED_STORE_LEVEL_DIRS_MEDIUMTYPE, 0)
-          .setDefaultValue("MEM")
-          .setDescription("The media type for the top storage tier directories ")
+          .setDefaultValue(
+              String.format("${%s}", PropertyKey.WORKER_TIERED_STORE_LEVEL0_ALIAS.mName))
```

Comment:
In other places we use `Name.PROPERTY_KEY`, could we be consistent?

Reply:
fixed